docs/try/tutorials.md

```diff
@@ -1,7 +1,7 @@
 
 #Tutorials
 ##Basics
- * [Install and Compile](): link to a webinar to compile and install RINGmesh.
+ * [Configure and Compile](tutorials/configure_compile_ringmesh): How to configure and compile the RINGMesh source code.
```

Comment:
I feel that we should not put "the" before "RINGMesh source code" but maybe we should ask to someone else.